Source/SwiftLintFrameworkTests/YamlParserTests.swift

```diff
@@ -0,0 +1,30 @@
+//
```

Comment:
This is mostly testing that the `YamlParserError` that I introduced works as expected. Would you like to not test this? Or even switch parsing to return an optional instead of throwing?